server/chat-plugins/chat-monitor.ts

```diff
@@ -667,8 +667,65 @@ export const commands: ChatCommands = {
 		}
 		this.globalModlog(`ALLOWNAME`, target);
 	},
+	testfilter(target, room, user, connection, cmd) {
+		this.privatelyCheckCan('lock');
+		this.checkBroadcast(true);
+		if (room && ['staff', 'upperstaff'].includes(room.roomid)) this.runBroadcast();
+		let [message, monitorName] = target.split('|').map(x => x.trim());
+		if (!monitorName && cmd !== 'testfilter') {
+			monitorName = cmd.slice(4);
+			if (!Chat.monitors[monitorName] && Chat.monitors[monitorName.replace('filter', '')]) {
+				monitorName = monitorName.replace('filter', '');
+			}
+		}
+		// namefilter doesn't have a monitor function
+		if (!(monitorName && message) || !Chat.monitors[monitorName]?.monitor) {
+			return this.parse(`${this.cmdToken}help testfilter`);
+		}
+		const monitor = Chat.monitors[monitorName].monitor!;
+		const lcMessage = Chat.stripFormatting(message
+			.replace(/\u039d/g, 'N').toLowerCase()
```

Comment:
```suggestion
			.replace(/\u039d/g, 'N')
			.toLowerCase()
```

Reply:
TBH this should probably be its own function.

Reply:
agree